guidelines/commit_and_pr_text.md

```diff
@@ -0,0 +1,56 @@
+# Motivation
+Our changelogs (reviewed by users/customers) are autogenerated from PR titles and git commit text.  The better this information is, the more we help our customers.  This information also helps reviewers understand changes, patch managers understand risk/importance, and when hunting for regressions, helps us better understand what is/isn’t likely to be a cause.
```

Comment:
I would add something about current and future team members trying to understand why things are they way they are, as well.  The commit log has the history of what we know about what was changed and why.

Reply:
added